source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/products/index.tsx

```diff
@@ -51,17 +86,15 @@ class ProductsRoute extends React.Component<IProps & i18nProps> {
 
         <hr />
 
-        <h3 className='p-b-lg'>{t('org.productSelectTitle')}</h3>
-
-        <p style={{ width: '200px', overflow: 'auto'}}>
-          TODO: render available products from DWKit as demonstrated
-          https://app.zeplin.io/project/5b3b85b95b0fc79a4b3c40c9/screen/5b3b8d851853ea0a2edf9ce0
-        </p>
+        <h3 className='p-b-md'>{t('org.productSelectTitle')}</h3>
+        <ProductDefinitionMultiSelect {...multiSelectProps} />
       </div>
     );
   }
 }
 
 export default compose(
-  translate('translations')
+  withTranslations,
+  withOrbit(mapRecordsToProps),
+  withLoader(({ organizationProductDefinitions }) => !organizationProductDefinitions),
```

Comment:
do we need a loader? won't all the records be loaded from the parent component already?